test/test_aiortc.sh

```diff
@@ -11,9 +11,14 @@ JANUS_SRC="$( dirname $SCRIPTPATH )"
 TEST=${1-"$SCRIPTPATH/echo.py"}
 URL=${2-"ws://localhost:8188/"}
 
-echo "Starting Janus binary from $JANUS_SRC ..."
-$JANUS_SRC/src/janus >/dev/null 2>&1 &
-JANUS_PID=$!
+JANUS_RUNNING=$(ps ax | grep "bin\/janus" | grep -v grep | wc -l)
```

Comment:
This seems convoluted and overly complicated.

I'd suggest using instead:
```bash
JANUS_RUNNING=$(pgrep -x janus)
JANUS_RUNNING=${JANUS_RUNNING:-0}
```

Reply:
What if you replace
```bash
JANUS_RUNNING=$(pgrep -x janus)
JANUS_RUNNING=${JANUS_RUNNING:-0}
```
with:
```bash
JANUS_RUNNING=$(pgrep -xc janus || true)
```

Reply:
thanks, that has worked in both Debian and Ubuntu.